users/kimurayoshimi/9.js

```diff
@@ -2,6 +2,10 @@ const a = (b) => b.includes("e");
 const b = a(["a", "b", "c"]);
 console.log(b);
 
-const c = (d) => d.every((value) => value === "e");
+const c = (d) => d.some((value) => value === "e");
 const d = c(["a", "b", "c"]);
 console.log(d);
+
+const e = (f) => f.find((value) => value === "e");
+const f = e(["a", "b", "c"]);
+console.log(f);
```

Comment:
The commit was not corrected correctly, and has been fixed.
## Run-time commands
```
node ./users/kimurayoshimi/9.js
```
## [stackblitz](https://stackblitz.com/edit/node-nf3np9?file=users%2Fkimurayoshimi%2F9.js)